lms/envs/common.py

```diff
@@ -365,7 +365,7 @@
     'ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES': True,
 
     # WIP -- will be removed in Ticket #TNL-4750.
-    'ENABLE_TIME_ZONE_PREFERENCE': False,
+    'ENABLE_TIME_ZONE_PREFERENCE': True,
```

Comment:
A good way to make sure it doesn't get merged in is to introduce a pylint error here.  I recommend adding an inline comment without a space before it:

    'ENABLE_TIME_ZONE_PREFERENCE': True,#FIXME: set this back to False